core/management/commands/load_copyright_map.py

```diff
@@ -0,0 +1,30 @@
+import os
+import logging
+import datetime
+
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+from core.load_copyright_map import loadCopyrightMap
+from core.management.commands import configure_logging
+
+configure_logging("load_copyright_map_logging.config", "load_coyright_map.log")
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Load copyrights maps"
```

Comment:
Can we get more help here?  We should be telling users more than what the command expects, what it changes in the app, and maybe an example of the command.  It's also probably worth noting that copyright maps which have been loaded cannot currently be deleted except via direct database queries.  People need to be 100% certain their stuff is good.